pyswarms/utils/search/base_search.py

```diff
@@ -73,7 +90,7 @@ def generate_score(self, options):
         ----------
 
         options: dict
-            a dict of 5 hyperparameter values ('c1', 'c2', 'w', 'k', 'p' 
+            a dict of 5 hyperparameter values ('c1', 'c2', 'w', 'k', 'p'
```

Comment:
Is a parenthesis missing here? 😄 ❓ 
I guess we can also rephrase this as:
```python
"""
a dict with the following keys: {'c1 ', 'c2', 'w', 'k', 'p'}
"""
```

Reply:
Hi @ljvmiranda921 

> Is a parenthesis missing here?

Sure enough. :smile: I was afraid to change that line, though -- it's not mine. I just appears in the diff because my vim is set to eliminate trailing spaces on save of a .py file. I'll rephrase it according to your suggestion.